spec/shoryuken/queue_registration_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+RSpec.describe Shoryuken::QueueRegistration do
+  describe "#register_queues!" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.